core/db/migrate/20250825175217_add_missing_page_builder_indexes.rb

```
@@ -0,0 +1,7 @@
+class AddMissingPageBuilderIndexes < ActiveRecord::Migration[8.0]
+  def change
+    add_index :spree_themes, %w[store_id default], name: 'index_spree_themes_on_store_id_and_default', if_not_exists: true
+    add_index :spree_page_links, %w[linkable_id parent_type parent_id], name: 'index_spree_page_links_on_linkable_id_and_parent_type_and_parent_id', if_not_exists: true
```

RuboCop (1.76.1)

[convention] Line is too long. [168/150]

(Layout/LineLength)